apps/src/code-studio/pd/teacher_application/teacher_application.jsx

```diff
@@ -555,6 +579,18 @@ const TeacherApplication = React.createClass({
       }
     });
 
+    if (this.validateDistrictData(formData)) {
```

Comment:
why not pass in `districtValues` instead of `formData`, since `formData` includes a bunch of other stuff.